getting_started.md

```diff
@@ -8,11 +8,15 @@ The goal of this document is get the site and backend working locally on your de
 
 ### Public Site
 
+Install `node >= 12.x.x` and `npm`: https://nodejs.org/en/download/.
+
+If you are on Linux or macOS you can install using either [n](https://github.com/tj/n) or [nvm](https://github.com/nvm-sh/nvm)
+
 From the root directory of this repo:
 
 ```
-$ npm install -g http-server
-$ http-server
+$ ./local_setup.sh
+$ npm run dev
```

Comment:
For me when I did this step, it started up on port 3000, so the instruction below about :8080 no longer applies?

Reply:
Update the doc.  Yep its now port 3000, which is the typical port used by node apps.